docker/run-local.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e
+
+cd /agora
+sbt "~reStart --- $JAVA_OPTS"
```

Comment:
Nice! I was referencing our existing code and also this new how-to doc: https://broadinstitute.atlassian.net/wiki/display/GAWB/How+to+pass+Java+options+to+sbt+~reStart